libr/include/r_util/pj.h

```diff
@@ -39,6 +39,8 @@ R_API PJ *pj_kd(PJ *j, const char *k, double d);
 R_API PJ *pj_kf(PJ *j, const char *k, float d);
 R_API PJ *pj_kb(PJ *j, const char *k, bool v);
 R_API PJ *pj_null(PJ *j);
+R_API PJ *pj_r(PJ *j, const unsigned char *k, size_t len);
+R_API PJ *pj_kr(PJ *j, const char *name, const unsigned char *k, size_t len);
```

Comment:
same here. `name` is actually the key AFAIU and `k` is actually the value, so it should be `v` and `len` -> `v_len`.